docs/UserGuide.md

```diff
@@ -14,6 +14,17 @@ guest management tasks done faster than traditional GUI apps.
 
 --------------------------------------------------------------------------------------------------------------------
 
+## Components of the UI
+
+![UiComponentLabels](images/UiComponentLabels.png)
+
```

Comment:
Should the "room cleaned" be "room clean status" or a better word to denote a status? Alternatively, we can consider putting marking red as "room not cleaned" and green as "room cleaned" in the diagram

Reply:
Agreed.